style.css

```diff
@@ -943,6 +1060,13 @@ form input:focus {
     width: 50%;
     margin: 0;
     padding-right: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    color: #262626;
```

Comment:
As suggested by the previous reviewer, kindly use the right font colour `#091E42` for the text in the project card
 Your design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/bbe3f1f7-b895-4adf-9d6e-e9407b7473aa)

Also, please use the right font-colour for the horizontal line after the about-me section

Your design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/71a42589-f027-4341-bcb7-a57e01a77755)

Figma design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/40b63fbb-66ed-4873-802a-6ba32c8ca624)

Reply:
Thank you so much for your feedback. I think the font color in the contact section, for the desktop version, in Figma is black. I changed it anyway because I like how it looks better